test/run_test.py

```diff
@@ -450,6 +489,135 @@ def get_selected_tests(options):
     return selected_tests
 
 
+def test_impact_of_file(filename):
+    """Determine what class of impact this file has on test runs.
+
+    Possible values:
+        TORCH - torch python code
+        CAFFE2 - caffe2 python code
+        TEST - torch test code
+        UNKNOWN - may affect all tests
+        NONE - known to have no effect on test outcome
+        CI - CI configuration files
+    """
+    parts = filename.split(os.sep)
```

Comment:
isn't `os.path.split(filename)` the more idiomatic way to do this?

Reply:
Welp no, os.path.split is hardcoded to a maxsplit of 1 :/